indexing-service/src/test/java/io/druid/indexing/common/task/SameIntervalMergeTaskTest.java

```diff
@@ -68,14 +69,13 @@
   private final CountDownLatch publishCountDown = new CountDownLatch(1);
   private final IndexSpec indexSpec;
   private final ObjectMapper jsonMapper;
-  private IndexIO indexIO;
 
   public SameIntervalMergeTaskTest()
   {
+    EmittingLogger.registerEmitter(new NoopServiceEmitter());
```

Comment:
Can you double check this? I commented this line and the test passed. BTW, why this test will not fail before you added this, and why the test case would enter the logic about making any alerts?